src/main/scss/components/_buttons.scss

```diff
@@ -75,6 +75,35 @@
   }
 }
 
+.jenkins-button--outline {
+  color: var(--btn-secondary-color) !important;
+  border: 2px solid var(--btn-secondary-border);
+  font-weight: bold;
```

Comment:
These props could be inherited from the base `jenkins-button` class.

Reply:
Well but the original yui outline button had been bold, but if you want we can change it but IMHO it makes more sense bold.
![image](https://github.com/user-attachments/assets/b5e3c8a4-d453-4f2d-915e-2170a1585452)
vs
![image](https://github.com/user-attachments/assets/b54ea5ed-e0bf-4c43-9d9b-7844365bb5c6)